packages/vaex-core/vaex/dataframe.py

```diff
@@ -4415,8 +4415,20 @@ def __getitem__(self, item):
             df._selection_masks[FILTER_SELECTION_NAME] = vaex.superutils.Mask(df._length_unfiltered)
             return df
         elif isinstance(item, (tuple, list)):
-            df = self.copy(column_names=item)
-            return df
+            # check if list or tuple contains types
+            if all([True if type(i)==type else False for i in item ]):
```

Comment:
`True if expression else False` can be replaced `expression`, also check pylint.